request/iframe.js

```diff
@@ -212,10 +213,11 @@ define([
 								createInput(x, val[i]);
 							}
 						}else{
-							if(!formNode[x]){
+							var n = query("input[name='"+x+"']");
+							if(n.indexOf() == -1){
```

Comment:
This is fine, though it feels a bit magical (not that large blocks of this existing codebase aren't the same, but I try to leave things clearer than they were before)... I would probably add a comment and explain that the lack of indexOf detects attributes vs. nodes.

Reply:
Done. Reviewing my own code, I added the formNode to scope the `query()` to the descendants of just the node under consideration.